fastlane/Fastfile

```diff
@@ -282,7 +285,56 @@ platform :ios do
   end
 end
 
+platform :android do
 
+  desc "Deploy app to playstore"
+  desc "Params:"
+  desc "stage: Application build environment"
+  lane :deploy do |options|
+    do_playstore(options)
+  end
+
+  desc "Generate or download key"
+  desc "Params:"
+  desc "force_download: true|false"
+  lane :keystore do |options|
+    app_id = app_id_for('production')
+    workspace = Match::GitHelper.clone(google_git_url, true)
+    keystore = File.join(workspace, "#{app_id}.keystore")
+    unless File.exist?(keystore)
+      # TODO: Get storepass and keypass from somewhere
+      sh "cd #{workspace}; keytool -genkey -v -keystore #{app_id}.keystore -dname \"CN=us.platan, OU=Platanus, O=Platanus, L=Santiago, S=, C=CL\" -alias alias_name -keyalg RSA -keysize 2048 -validity 10000 -storepass holahola -keypass holahola"
+      message = Match::GitHelper.generate_commit_message({app_identifier: app_id, type: 'keystore'})
+      Match::GitHelper.commit_changes(workspace, message, google_git_url)
+    end
+    keystore
+  end
+
+  private_lane :do_playstore do |options|
+    build(options)
+    sign(options)
+    zipalign(options)
+  end
+
+  private_lane :build do |options|
+    stage = options[:stage]
+    sh "cordova prepare --env=#{stage}"
+    sh "cd ..; ionic build android --env=#{stage} --release"
```

Comment:
Because of this: https://github.com/fastlane/fastlane/issues/155

`sh` runs inside `./fastlane` and while cordova works `cordova` works everywhere inside the project, `ionic` doesn't